_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,188 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { getCurrencyDefaults } from '@automattic/format-currency';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ExternalLink from 'components/external-link';
+
+import './single-product-backup.scss';
+
+export function SingleProductBackup( props ) {
+	const { products, siteRawUrl } = props;
+
+	const backupPlanPrices = {
+		jetpack_backup_daily: {
+			monthly: products.jetpack_backup_daily_monthly.cost,
+			yearly: products.jetpack_backup_daily.cost,
+		},
+		jetpack_backup_realtime: {
+			monthly: products.jetpack_backup_realtime_monthly.cost,
+			yearly: products.jetpack_backup_realtime.cost,
+		},
+	};
+
+	const currencySymbol = getCurrencyDefaults( products.jetpack_backup_daily.currency_code ).symbol;
+
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	return (
+		<React.Fragment>
+			<h1 className="plans-section__header">{ __( 'Single Products' ) }</h1>
+			<h2 className="plans-section__subheader">
+				{ __( "Just looking for backups? We've got you covered." ) }
+			</h2>
+			<div className="single-product-backup__accented-card-container">
+				<div className="single-product-backup__accented-card">
+					<div className={ 'single-product-backup__accented-card__header' }>
+						<SingleProductBackupHeader
+							currencySymbol={ currencySymbol }
+							backupPlanPrices={ backupPlanPrices }
+						/>
+					</div>
+					<div className={ 'single-product-backup__accented-card__body' }>
+						<SingleProductBackupBody
+							currencySymbol={ currencySymbol }
+							selectedBackupType={ selectedBackupType }
+							setSelectedBackupType={ setSelectedBackupType }
+							backupPlanPrices={ backupPlanPrices }
+							siteRawUrl={ siteRawUrl }
+						/>
+					</div>
+				</div>
+			</div>
+		</React.Fragment>
+	);
+}
+
+function SingleProductBackupHeader( props ) {
+	const { currencySymbol, backupPlanPrices } = props;
+
+	return (
+		<div className="single-product-backup__header-container">
+			<h3>{ __( 'Jetpack Backup' ) }</h3>
+			<PlanPriceDisplay currencySymbol={ currencySymbol } backupPlanPrices={ backupPlanPrices } />
+		</div>
+	);
+}
+
+export function PlanPriceDisplay( props ) {
```

Comment:
I put each component in its own PR except for `SingleProductBackupHeader`, which I left here since it is pretty small.

https://github.com/Automattic/jetpack/pull/14025
https://github.com/Automattic/jetpack/pull/14026
https://github.com/Automattic/jetpack/pull/14027/